### PR TITLE
Add etcd2-proxy-proxy as etcd2 replacement

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: template/coreos/cloud-config-node.yaml [hunk 1]
```diff
@@ -7,34 +7,62 @@ ssh_authorized_keys:
...
+      command: start
+      content: |
+        [Unit]
+        Description=etcd2-proxy-proxy downloader
+
+        [Service]
+        Type=oneshot
+        RemainAfterExit=yes
+        ExecStartPre=/usr/bin/mkdir -p /opt/bin
+        ExecStart=/bin/bash -c "test -x /opt/bin/etcd2-proxy-proxy || /usr/bin/curl -L -o /opt/bin/etcd2-proxy-proxy https://s3-eu-west-1.amazonaws.com/kubermatic/coreos/etcd2-proxy-proxy && /usr/bin/chmod +x /opt/bin/etcd2-proxy-proxy"
```
Comment (s-urbaniak) [inline template/coreos/cloud-config-node.yaml:45]: we should do the same for the kubelet and socat

Comment (sttts) [inline template/coreos/cloud-config-node.yaml:45]: Maybe we do a downloader unit.

File: template/coreos/cloud-config-node.yaml [hunk 2]
```diff
@@ -7,34 +7,62 @@ ssh_authorized_keys:
 coreos:
   flannel:
     interface: $private_ipv4
-    etcd_endpoints: "{{.EtcdURL}}"
```
Comment (s-urbaniak) [inline template/coreos/cloud-config-node.yaml:10]: it defaults to localhost?

File: template/coreos/cloud-config-node.yaml (hunk 1 above)
Comment (sttts) [inline template/coreos/cloud-config-node.yaml:45]: But in the next PR.

Comment (s-urbaniak) [inline template/coreos/cloud-config-node.yaml:45]: :+1:

**Comment by s-urbaniak** [comment]
lgtm

File: template/coreos/cloud-config-node.yaml (hunk 2 above)
Comment (sttts) [inline template/coreos/cloud-config-node.yaml:10]: Yes.

**Pull request merged**